### PR TITLE
misc/PureSerialization

### PR DESCRIPTION
**Pull request by dmcyk**
This is one hell of a cleanup. Basically just ditched the builtin networking layer and two dependencies together with it - `Result` and `BrightFutures`. 

Seems like `watchOS` target was the only one out of date not updated for Swift 4, so I did that so.

### CODE REVIEW

File: Spine/Spine.swift [hunk 1]
```diff
@@ -49,349 +37,9 @@ open class Spine {
 	/// - parameter networkClient: The NetworkClient to use
 	///
 	/// - returns: The Spine instance.
-	public init(router: Router, networkClient: NetworkClient) {
-		self.router = router
-		self.networkClient = networkClient
-		self.operationQueue.name = "com.wardvanteijlingen.spine"
```
Comment (foadsaliba) [inline Spine/Spine.swift]: lol

**Comment by dmcyk** [comment]
nopee, at least for serialisation, that was all for networking 😁

**Comment by foadsaliba** [comment]
And nothing you deem useful for us? If useful we could use the operations instead of a promises?

**Comment by dmcyk** [comment]
The only generally useful operation there is the [ConcurrentOperation](https://github.com/discovery-fusion/dkids_apple_spine/blob/master/Spine/Operation.swift#L52), it doesn't give much though, only some utility for state management. 

Other operations were concrete tasks used for the network layer. Even if we were to do our approach on operations I guess we'd de better off just defining them in SonicKit itself.

**Comment by dmcyk** [comment]
just added the old `0.8.0` and bumped version in the main spec to `0.9.0` once merged I'll tag this :D I'm not forgetting anything, right? Pods are so 2017 😂

**Comment by foadsaliba** [comment]
you need to add a new versioned one in the Specs folder under a 0.9.0 folder.
the podspecs in Specs folder are used when you do a source and then specify a version.
The root podspec is used when you use `":git =>"` syntax

**Pull request merged**